Assignments/stu1/assignments.py

```diff
@@ -182,4 +183,8 @@ def read_message_from_sqs(message):
     return retval
 
 def save_to_cat_table(cat):
```

Comment:
This function needs to save a cat to the cat table (as the name implies: `save_to_cat_table`).